.circleci/config.yml

```diff
@@ -59,6 +60,26 @@ commands:
             --excludePackages 'uuid@2.0.1'
             --excludePrivatePackages # uuid@2.0.1 has an issue with the license information but it is licensed under MIT
 
+  upload_test_results_and_coverage:
+    parameters:
+      base_flag:
+        type: enum
+        enum: ["sdk", "dapp", "cli"]
+      test_kind_flag:
+        type: enum
+        enum: ["unit", "integration", "e2e"]
+    steps:
+      - store_test_results:
+          path: reports/junit
+      - store_artifacts:
+          path: reports/junit
+      - run:
+          name: "Upload test coverage results to CodeCov"
+          command: >
+            bash <(curl -s https://codecov.io/bash) -C $CIRCLE_SHA1
+            -F << parameters.base_flag >>
+            -F << parameters.base_flag >>.<< parameters.test_kind_flag >>
```

Comment:
What do the double flags do? Make it so that I can filter for all sdk tests?

Reply:
First this feature was not added, but was there before. Seconds yes, this makes it easier to identify coverage changes by "location". See for example the table in [this report](https://github.com/raiden-network/light-client/pull/2291#issuecomment-716520206).